Rakefile

```diff
@@ -4,8 +4,13 @@ rescue LoadError
   puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
 end
 
-APP_RAKEFILE = File.expand_path("../spec/internal/Rakefile", __FILE__)
-load 'rails/tasks/engine.rake'
+# While I am not sure, if rails' engine rake tasks are usuable with this app
```

Comment:
The only thing we do is extend some of the Rails rake tasks, but I don't think we need to actually load the rails tasks to do that. See https://github.com/culturecode/stagehand/blob/88a0f9ae748d9ef8bbee51e13617650a546c5daf/lib/tasks/stagehand_tasks.rake#L36-L38.